activiti-spring-boot-starter/src/test/resources/connectors/variable-mapping.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "variableMappingConnectorId",
+  "name": "Name of the connector",
+  "description": "Description of the connector",
+  "icon": "icon-name",
+  "actions": {
+    "variableMappingActionId": {
+      "id": "variableMappingActionId",
+      "name": "variableMappingActionName",
+      "description": "...",
+      "input": [
```

Comment:
plural inputs and outputs: from https://github.com/Activiti/Activiti/blob/develop/activiti-spring-boot-starter/src/test/resources/connectors/process-image.json#L11
and rfc doc